mvc/action-return-types/7.x/WebApiSample/Controllers/ProductsController.cs

```diff
@@ -0,0 +1,55 @@
+namespace WebApiSample.Controllers;
+
+using Microsoft.AspNetCore.Mvc;
+using WebApiSample.Models;
+using WebApiSample.Repositories;
+
+[ApiController]
+[Route("[controller]")]
+public partial class ProductsController : ControllerBase
```

Comment:
The original sample code has all the Actions in the same file, to keep it easier to read I move them to multiple files. So, this change will break all the sample code references, in docs.